src/github/create-commit.ts

```diff
@@ -44,6 +60,9 @@ export async function createCommit(
       message,
       tree: treeSha,
       parents: [refHead],
+      signature,
```

Comment:
If you can find the REST documentation in the GitHub API, adding it here as a comment would be useful.

I'm interested to know what options we have available to us for a signing algorithm (I'm hoping we might be able to use something other than GPG).

Reply:
Added a link to the create commit API docs. It looks like only PGP is supported for this API